src/server/twitterSearch.js

```diff
@@ -15,51 +16,161 @@ var T = new Twit({
   'timeout_ms':           60 * 1000,  // optional HTTP request timeout to apply to all requests.
 });
 
-const makeTweeterFromRaw = (raw) => {
-  return Builders.TweeterBuilder()
-    .id(raw.id)
-    .name(raw.name)
-    .handle(raw.screen_name)
+/**
+ * Convert some raw status from the Twitter API into a proper immutable Tweet object.
+ * @param rawTweet From the Twitter API.
+ * @returns {ImmutableTweet}
+ */
+const buildTweetFromRaw = (rawTweet) => {
+  return Builders.TweetBuilder()
+    .id(rawTweet.id)
+    .content(rawTweet.text)
+    .date(moment(new Date(rawTweet.created_at)).format('YYYY-MM-DD HH:mm:ss'))

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'created_at' is not in camel case.](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2175945336/issues/source?bid=3148782&fileBranchId=3163167#l28)